# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLContext.h
# Commit: 5005fe56b7b0
# Full Hash: 5005fe56b7b0153d2069d25c6f1f4e311e2c1eb1
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-02-21 05:56:31
# Regressor Bug: 1616715
# File Overlap Count: 1
# Description:
#   Bug 1616715 - SurfaceFromElement gets alpha Premult unless opt-in to NonPremult. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D63421
# ==============================================================================

diff -r 985dccd11e97 -r 5005fe56b7b0 dom/canvas/WebGLContext.h
--- a/dom/canvas/WebGLContext.h	Thu Feb 20 21:12:05 2020 +0000
+++ b/dom/canvas/WebGLContext.h	Thu Feb 20 20:52:10 2020 +0000
@@ -374,9 +374,6 @@
 
   void SetCompositableHost(RefPtr<layers::CompositableHost>& aCompositableHost);
 
-  RefPtr<mozilla::gfx::SourceSurface> GetSurfaceSnapshot(
-      gfxAlphaType* out_alphaType);
-
   /**
    * An abstract base class to be implemented by callers wanting to be notified
    * that a refresh has occurred. Callers must ensure an observer is removed